packages/vue/src/components/URLParamsProvider.jsx

```diff
@@ -61,7 +61,6 @@ const URLParamsProvider = {
 					);
 				} catch (e) {
 					// Do not set value if JSON parsing fails.
-					console.error(e);
```

Comment:
@royduin We better change to `console.warn(e);`

Reply:
I don't think we need a message at all just like in the ReactiveBase. If it fails it's due a non reactivesearch param which should be catched and ignored.